addons/mail/static/src/components/discuss_sidebar/discuss_sidebar.js

```diff
@@ -175,13 +186,26 @@ class DiscussSidebar extends Component {
             discussIsAddingChannel: discuss && discuss.isAddingChannel,
             discussIsAddingChat: discuss && discuss.isAddingChat,
             discussSidebarQuickSearchValue: discuss && discuss.sidebarQuickSearchValue,
+            sendSound: mailRtc.sendSound,
+            sendVideo: mailRtc.sendVideo,
```

Comment:
Don't make an explicit distinction between sound and video, see discord, any user can enable or disable his video in any voice room.